go-sdk/dashboard/dashboard_test.go

```diff
@@ -110,32 +110,13 @@ func TestDashboardBuilder(t *testing.T) {
 	outputJSONFilePath := filepath.Join("..", "..", "internal", "test", "dac", "expected_output.json")
 	expectedOutput, readErr := os.ReadFile(outputJSONFilePath)
 
-	testSuites := []struct {
-		title          string
-		sdkResult      string
-		expectedResult string
-		expectedError  bool
-	}{
-		{
-			title:          "full dashboard",
-			sdkResult:      string(builderOutput),
-			expectedResult: string(expectedOutput),
-		},
-	}
-	for i := range testSuites {
-		test := testSuites[i]
-		t.Run(test.title, func(t *testing.T) {
-			fmt.Println(string(expectedOutput))
-			fmt.Println(string(builderOutput))
+	t.Run("classic dashboard", func(t *testing.T) {
```

Comment:
the TestDashboardBuilder() code above is missing the changes you put in the PR description, isn't it?